data/mods/Magiclysm/achievements/statistics.json

```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": "avatar_levels_spell",
+    "type": "event_transformation",
+    "event_type": "character_levels_spell",
+    "value_constraints": { "character": { "equals_statistic": "avatar_id" } }
+  },
+  {
+    "id": "avatar_learns_spell",
+    "type": "event_transformation",
+    "event_type": "character_learns_spell",
+    "value_constraints": { "character": { "equals_statistic": "avatar_id" } },
+    "drop_fields": [ "spell" ]
+  },
+  {
+    "id": "num_spells_learned",
+    "type": "event_statistic",
+    "stat_type": "count",
+    "event_transformation": "avatar_learns_spell",
+    "description": { "str": "learned a spell", "str_pl": "number of spells learned" }
+  },
+  {
+    "id": "avatar_forgets_spell",
+    "type": "event_transformation",
+    "event_type": "character_forgets_spell",
+    "value_constraints": { "character": { "equals_statistic": "avatar_id" } },
+    "drop_fields": [ "spell" ]
+  },
+  {
+    "id": "num_spells_forgotten",
+    "type": "event_statistic",
+    "stat_type": "count",
+    "event_transformation": "avatar_forgets_spell",
+    "description": { "str": "forgot a spell", "str_pl": "number of spells forgotten" }
```

Comment:
These descriptions are always formatted after either a number or a `x/y`, so probably "spell forgotten" and "spells forgotten" are better descriptions (similarly for the other stats).